### PR TITLE
Develop Hello world front-end

### PR DESCRIPTION
**Pull request by raihan2bd**
Hi There 👋 ,
### In this branch, I made the following changes ⬇️ 
- Setup the project with the correct Linters
- Removed unwanted files from the project
- Setup necessary routes to display the home and random greeting page
- Setup Redux sore with necessary reducer's actions to get random greetings from the back-end
- Added Home and Greeting components to display those pages
- Improved code quality

## Note: This is the front-end pull request. Here is the [BackEnd PR](https://github.com/raihan2bd/hello-world-back-end/pull/1). If you want to run this repo on your local machine make sure you run this client repo on port `3456`

***Thanks for visiting 👍. If you have any questions feel free to ask 🙂***

### CODE REVIEW

File: src/components/Greeting.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <h1>Random Greeting</h1>
+      <h3>{greeting}</h3>
+      <Link to="/">
+        <button type="button">Back to Home</button>
+      </Link>
+    </section>
+  );
+};
+
+export default Greeting;
```
Comment (tresorsawasawa) [inline src/components/Greeting.js:24]: - [x] As required for the project your two apps to work together, kindly ensure that your frontend app displays a random greeting to the UI. Right now, it shows a blank page and shows an error in your console. Below, is the image of how it looks like:
Screenshot from 2023-07-11 15-40-48

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @raihan2bd 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Greeting.js (hunk 1 above)
Comment (raihan2bd) [reply src/components/Greeting.js:24]: Hi @tresorsawasawa 👋,  Thanks for your information, However, everything is working on my side
Capture
Make sure your front-end port is `3456`
Capture2

Comment (tresorsawasawa) [reply src/components/Greeting.js:24]: Hi @raihan2bd 

If it is now working, that is good then.

**Review by Gambit142 — APPROVED** [review]
Hi @raihan2bd 👋,

## Status: Approved 🟢 

### Great work configuring Rails Cors 👏 🎉   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linter checks passed ✔️ 
- Configured backend properly ✔️ 
- Configured frontend properly ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**